backend/src/main/kotlin/org/loculus/backend/model/SubmitModel.kt

```diff
@@ -132,6 +150,9 @@ class SubmitModel(
         uploadDatabaseService.deleteUploadData(uploadId)
     }
 
+    /**
+     * Inserts the uploaded metadata (and sequence data) into the 'aux' tables in the database.
+     */
```

Comment:
Good comment, when I saw the function name on its own I had no clue what it was doing. We should probably improve the name to something like `insertDataIntoAux`